chain/chain/src/metrics.rs

```diff
@@ -4,19 +4,23 @@ use near_metrics::{
 };
 use once_cell::sync::Lazy;
 
-pub static BLOCK_PROCESSED_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
-    try_create_int_counter("near_block_processed_total", "Total number of blocks processed")
-        .unwrap()
+pub static BLOCK_PROCESSING_ATTEMPTS_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
+    try_create_int_counter(
+        "near_block_processing_attempts_total",
+        "Total number of block processing attempts.",
+    )
+    .unwrap()
 });
-pub static BLOCK_PROCESSED_SUCCESSFULLY_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
+pub static BLOCK_PROCESSED_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
     try_create_int_counter(
-        "near_block_processed_successfully_total",
+        "near_block_processed_total",
```

Comment:
I really tried finding the usage of this metric in grafana, and found none. Can't claim that my search was exhaustive, it's hard to click on every graph of every dashboard :)

Preserving the history of a metric is usually a good idea. In this case losing history should be acceptable, because I haven't found a single use case for the original metric.